server/functions/index.js

```diff
@@ -87,13 +114,12 @@ exports.sendNotificationNearbyOnNewFavor = functions.firestore
                     var latUser = user.location.latitude;
                     var longUser = user.location.longitude;
                     var distance = distanceInKm(longFav, latFav, longUser, latUser);
-                    if (distance < maxDistance && user.id !== posterId) {
+                    if (distance < user.notificationRadius && user.id !== posterId && user.activeAcceptingFavors === 0 && user.activeRequestingFavors === 0) {
```

Comment:
Do you think that those who have requested but haven't accepted should be notified as well? Even If a person has requested a favor, i think it wouldn't be bad to be notified about new favors around

Reply:
I agree, I changed it